scipy/linalg/decomp_svd.py

```diff
@@ -73,8 +73,10 @@ def svd(a, full_matrices=True, compute_uv=True, overwrite_a=False,
     Examples
     --------
     >>> from scipy import linalg
+    >>> from numpy.random import default_rng
+    >>> rng = default_rng()
```

Comment:
One more possible tweak is to omit the import and use `np.random.default_rng()` in the examples. Up to you.

Reply:
I think it would make the line a bit long then. If you don't mind I will let this like that.